packages/shared/ReactVersion.js

```diff
@@ -1,15 +1 @@
-/**
- * Copyright (c) Meta Platforms, Inc. and affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-// TODO: this is special because it gets imported during build.
-//
-// It exists as a placeholder so that DevTools can support work tag changes between releases.
-// When we next publish a release, update the matching TODO in backend/renderer.js
-// TODO: This module is used both by the release scripts and to expose a version
-// at runtime. We should instead inject the version number as part of the build
-// process, and use the ReactVersions.js module as the single source of truth.
-export default '19.0.0';
+export default '19.0.0-rc-d6cb4e77-20240911';
```

Comment:
revert

Reply:
Done in https://github.com/facebook/react/pull/30951/commits/c8f573e554d019af6db44062b9d2f00ede67d5c3